connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -270,8 +269,7 @@ private synchronized void recordSent(final ProducerRecord<byte[], byte[]> record
             removed = outstandingMessagesBacklog.remove(record);
         // But if neither one had it, something is very wrong
         if (removed == null) {
-            log.error("CRITICAL Saw callback for record that was not present in the outstanding message set: "
-                    + "{}", record);
+            log.error("{} CRITICAL Saw callback for record that was not present in the outstanding message set: {}", this, record);
```

Comment:
slf4j doesn't have `FATAL` by default. You have to do something like the following to do something like it:

https://github.com/apache/kafka/pull/3477/files#diff-683987e02eeaa80f76d5b965d15be393R134